package.json

```diff
@@ -42,8 +42,8 @@
     ]
   },
   "dependencies": {
-    "@actions/core": "^1.10.0",
-    "@actions/github": "^5.1.1",
```

Comment:
❓ Just confirming there were no breaking changes in this update?

Reply:
ill have a look, but from testing it seems fine 👀

Reply:
@trend-adam-kinnell only that it dropped support for node 14 & 16
<img width="528" alt="image" src="https://github.com/cloudconformity/npm-version-action/assets/89753679/920ba91f-1bd4-487e-b808-346fb8ec6f51">

Reply:
reverted this package to `^5.1.1` to not potentially break builds for now